src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -46,7 +43,7 @@ The most common causes of a poor CLS are:
 **Summary:** Always include `width` and `height` size attributes on your images and video elements. Alternatively, reserve the required space with [CSS aspect ratio boxes](https://css-tricks.com/aspect-ratio-boxes/). This approach ensures that the browser can allocate the correct amount of space in the document while the image is loading.
 
   <figure class="w-figure">
-    <video controls loop muted class="w-screenshot" poster="video-image-dimensions.jpg">
+    <video controls loop muted class="w-screenshot" poster="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/8wKRITUkK3Zrp5jvQ1Xw.jpg" | imgix }}">
```

Comment:
Should we use the Video shortcode here?

Reply:
We can, I'm downloading and manually uploading all of these videos, but it's coming along.